File: csharp/UrakawaToolkit/events/ChannelsPropertyEvent.cs
using System;
using System.Collections.Generic;
using System.Text;
using urakawa.property.channel;

namespace urakawa.events
{
	public class ChannelsPropertyEvent : PropertyEventArgs
	{
		public ChannelsPropertyEvent(ChannelsProperty src)
			: base(src)
		{
			SourceChannelsProperty = src;
		}

		public readonly ChannelsProperty SourceChannelsProperty;
	}
}

File: java/core/src/org/daisy/urakawa/events/property/channel/ChannelsPropertyEvent.java
package org.daisy.urakawa.events.property.channel;

import org.daisy.urakawa.events.property.PropertyEvent;
import org.daisy.urakawa.property.channel.IChannelsProperty;

/**
 * 
 *
 */
public class ChannelsPropertyEvent extends PropertyEvent {
	/**
	 * @param src
	 */
	public ChannelsPropertyEvent(IChannelsProperty src) {
		super(src);
		mSourceChannelsProperty = src;
	}

	private IChannelsProperty mSourceChannelsProperty;

	/**
	 * @return prop
	 */
	public IChannelsProperty getSourceChannelsProperty() {
		return mSourceChannelsProperty;
	}
}
